src/node/nodetonode.h

```diff
@@ -75,7 +76,12 @@ namespace ccf
       auto& n2n_channel = channels->get(t.from_node);
 
       if (!n2n_channel.verify(hdr, asCb(t)))
-        throw std::logic_error("Invalid authenticated node2node message");
+      {
+        throw std::logic_error(fmt::format(
```

Comment:
Could we add the same for `recv_encrypted()`?

Reply:
Certainly